vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -4,183 +4,184 @@ sidebarDepth: 2
 NGINX is working correctly if a `404 Not Found` error page is generated when accessing the EXTERNAL-IP from your browser. Alternatively, you can [set up a simple test application](../devops/manage-nginx.md#verify-the-nginx-ingress-install) using your local `kubectl`. You can also [customize the NGINX ingress](../devops/manage-nginx.md#customize-the-nginx-configuration) to optimize the configuration for Entando.
 :::
+
+See the K8s [Load Balancer Install Guide for AWS](https://kubernetes.github.io/ingress-nginx/deploy/#aws) and the [NGINX as Ingress Controller on EKS](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more details.
+
 ### Install the Entando Custom Resource Definitions
-Once per cluster you need to deploy the `Entando Custom Resources`.
-1. Download the Custom Resource Definitions and deploy the cluster scoped resources
-```sh
-kubectl apply -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/cluster-resources.yaml
+Once per cluster, deploy the `Entando Custom Resources`.
+1. Download the Custom Resource definitions and deploy the cluster scoped resources:
 ```
-Next you can create a namespace for the Entando Application.
-
-2. Create the namespace
+kubectl apply -f https://raw.githubusercontent.com/entando-k8s/entando-k8s-operator-bundle/v7.0.0-pre5/manifests/k8s-116-and-later/namespace-scoped-deployment/cluster-resources.yaml
+```
+2. Create the namespace for the Entando Application:
 ```sh
 kubectl create namespace entando
 ```
-3. Install the namespace scoped resources
-```sh
-kubectl apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v6.3.2/dist/ge-1-1-6/namespace-scoped-deployment/orig/namespace-resources.yaml
+3. Download the ConfigMap template as a starting point:
 ```
-If you run `kubectl -n entando get pods`, you'll see the Entando operator is now running.
+curl -sfL "https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/samples/entando-operator-config.yaml"
+```
+4. Edit the `entando-operator-config.yaml` to modify existing settings or add new ones. 
 
-## Deploy Your Entando Application
-You can now deploy your application to Amazon EKS.
-1. Download and unpack the `entando-helm-quickstart` release:
+5. Apply the ConfigMap: 
 ```sh
-curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.3.2.tar.gz | tar xvz
+kubectl apply -f entando-operator-config.yaml -n entando
 ```
-- See the included README file for more information on the following steps.
-2. Go to the downloaded directory
+
+5. Install the namespace scoped resources
 ```sh
-cd entando-helm-quickstart-6.3.2
+kubectl apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/namespace-scoped-deployment/namespace-resources.yaml
+```
+5. Run `kubectl get pods -n entando --watch` to see the pods start up. Use Ctl-C to exit.
```

Comment:
idk if you wanted to adopt `Ctl-C` or if we should discuss formatting for keystrokes?